### PR TITLE
Tabs for dashboard and email filter

### PR DESCRIPTION
**Pull request by mehalshah**
WITNESS THE GLORY OF TABS AND EMAIL FILTERING

### CODE REVIEW

File: apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
+
+  componentWillMount() {
+    this.getFilteredResults = _.debounce(this.getFilteredResults, 1000);
+    this.setState({submissions: this.props.submissions});
+  }
+
+  handleTeacherEmailChange = (event) => {
+    this.setState({email_filter: event.target.value});
+
+    // Find something to do email regex
```
Comment (clareconstantine) [inline apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx]: do you still need this comment?

File: apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx [hunk 2]
```diff
@@ -1,12 +1,55 @@
...
+    this.getFilteredResults();
+  }
+
+  getFilteredResults() {
+    this.loadRequest = $.ajax({
+      method: 'GET',
+      url: `/api/v1/peer_review_submissions/index?filter=escalated&email=${this.state.email_filter}`,
```
Comment (clareconstantine) [inline apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx]: shouldn't the `filter` param be dynamically set in the request depending on what tab you are on?

Comment (mehalshah) [reply apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx]: Ooo good catch. Will fix

File: apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx (hunk 1 above)
Comment (mehalshah) [reply apps/src/code-studio/peer_reviews/PeerReviewSubmissions.jsx]: Nope

**Pull request merged**